certbot.wrapper

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+join() {
+    sep=$1
+    first=$2
+    if [ "$first" != "" ]; then
+        shift 2
+        echo -n "${first}"
+        for item in "$@"; do echo -n "${sep}${item}"; done
+        echo
+    fi
+}
+
+paths=$(for plugin_snap in $(snap connections certbot|sed -n '2,$p'|awk '$1=="content[certbot-1]"{print $3}'|cut -d: -f1); do echo /snap/$plugin_snap/current/lib/python3.6/site-packages; done)
+export PYTHONPATH=$PYTHONPATH:$(join : $paths)
```

Comment:
I'm not sure if this matters, but if `PYTHONPATH` is initially empty, its value after this line will start with a colon. Can we either make sure this doesn't matter or fix the value in this case? This may not apply here, but [sys.path docs](https://docs.python.org/3/library/sys.html#sys.path) for example talk about how an empty value means to search in the current working directory for Python modules to import.

Reply:
Not sure if it would matter either, but happy to make this change.